### ISSUE TITLE
Different icons for "read" and "sent"

### ISSUE THREAD

**Issue by Mbuni21** [FossifyOrg/Messages#126]
### Checklist

- [X] I made sure that there are **no existing issues** - [open](https://github.com/FossifyOrg/Messages/issues) or [closed](https://github.com/FossifyOrg/Messages/issues?q=is%3Aissue+is%3Aclosed) - to which I could contribute my information.
- [X] I made sure that there are **no existing discussions** - [open](https://github.com/FossifyOrg/Messages/discussions) or [closed](https://github.com/FossifyOrg/Messages/discussions?discussions_q=is%3Aclosed) - to which I could contribute my information.
- [X] I have read the FAQs inside the app (Menu -> About -> FAQs) and my problem isn't listed.
- [X] **I have taken the time to fill in all the required details. I understand that the bug report will be dismissed otherwise.**
- [X] This issue contains only one feature request.
- [X] I have read and understood the [contribution guidelines](https://github.com/FossifyOrg/Messages/blob/master/CONTRIBUTING.md).
- [X] I optionally [donated](https://fossify.org/donate) to support the Fossify mission.

### Feature description

When you send a message, it gets one tick, when the person received it, it gets a second tick (if I understood it correctly :=) )
It would be cool if there were a symbol / sign to see if the person read it, i.e. a reading confirmation.

### Why do you want this feature?

In order to see if my recipient read the message, or if it even arrived.

### Additional information

A optional sign would be a sort of eye-shaped circle. (similar to Microsoft Teams style)

**Comment by Aga-C** [comment]
> when the person received it, it gets a second tick (if I understood it correctly :=) ) 

Yes, the second tick tells if the message has arrived to the recipient. 

> It would be cool if there were a symbol / sign to see if the person read it, i.e. a reading confirmation.

There's no reading confirmation in SMS. It's only available in RCS, but [we don't support it](https://github.com/FossifyOrg/Messages/discussions/34).

**Comment by Mbuni21** [comment]
Okay, thanks for the info! :)
